src/main/resources/archetype-resources/src/main/assembly/bin.xml

```diff
@@ -6,7 +6,7 @@
 	xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0 http://maven.apache.org/xsd/assembly-1.1.0.xsd">
-	<id>${symbol_dollar}{project.version}</id>
```

Comment:
Making the ID equal to the project version is something I did earlier, but it is really not convenient and leads to the version being repeated in the `tar.gz` name.